catppuccin_options_tmux.conf

```diff
@@ -3,6 +3,80 @@
 # This is executed separately to the main configuration
 # so that options are set before parsing the rest of the config.
 
+# Reset everything if requested.
+#
+# Useful for auto switching between themes:
+#
+# set-hook -g client-dark-theme {
+#   set -g @catppuccin_flavor "frappe"
+#   set -g @catppuccin_reset "true"
+#   run ~/code/github.com/catppuccin/tmux/catppuccin.tmux
+# }
+# set-hook -g client-light-theme {
+#   set -g @catppuccin_flavor "latte"
+#   set -g @catppuccin_reset "true"
+#   run ~/code/github.com/catppuccin/tmux/catppuccin.tmux
+# }
```

Comment:
Great work on this. That patch you linked to is exciting. I'm not sure when it's planned for release though. Do you think these should be configurable so users can set the path and the flavor? I wonder if we should document the `set-hook` functionality instead of incorporating it. But I'm also not super familiar with the feature yet.

Reply:
What should be configurable? These are comments

Also hooks are already documented, unless there's something special here worth mentioning? https://man7.org/linux/man-pages/man1/tmux.1.html#HOOKS

Reply:
Ah, yeah I think I missed that the first time. I think the example can stay as a comment then.